File: hw/Rodriguez/u2/hw17/ComplexOperation1/ComplexOperation/ComplexOperation/src/main/java/ec/edu/espe/complexoperation/view/ComplexOperation.java
package ec.edu.espe.complexoperation.view;

import ec.edu.espe.complexoperation.controller.Operation;
import ec.edu.espe.complexoperation.controller.PersonController;
import java.time.LocalDate;

/**
 *
 * @author David Rodriguez,THEJAVABANDITS DCCO-ESPE
 */
public class ComplexOperation {

    public static void main(String[] args) {
        System.out.println("Welcome to Complex Operation! \nDavid Rodriguez");

        float addend1;
        float addend2;
        float sum;

        //keyboar imput
        addend1 = 1;
        addend2 = 2;
        sum = Operation.addTwoNumbers(addend1, addend2);
        OperationPrinting.printAdditon(addend1, addend2, sum);

        addend1 = 1.2F;
        addend2 = 2.4F;
        sum = Operation.addTwoNumbers(addend1, addend2);
        OperationPrinting.printAdditon(addend1, addend2, sum);

        addend1 = 2.2F;
        addend2 = -1.8F;
        sum = Operation.addTwoNumbers(addend1, addend2);
        OperationPrinting.printAdditon(addend1, addend2, sum);
        
        LocalDate birthDate = LocalDate.parse("2004-11-10");
        int days;
        int years;
        
        days = PersonController.computeAgeInDays(birthDate);
        years = PersonController.computeAgeInYears(birthDate);
        System.out.println("David is " + days + " days old");
        System.out.println("He was born on " + birthDate);
        System.out.println("David is " + years + " year old");
    }
}

File: hw/Rodriguez/u2/hw17/ComplexOperation/ec_edu_espe_complexoperation_view/complex_operation.py
# complex_operation.py
from ec_edu_espe_complexoperation_controller.operation import Operation
from ec_edu_espe_complexoperation_controller.person_controller import PersonController
from ec_edu_espe_complexoperation_view.operation_printing import OperationPrinting
from datetime import date

def main():
    print("Welcome to Complex Operation! \nDavid Rodriguez")

    addend1 = 1
    addend2 = 2
    sum = Operation.add_two_numbers(addend1, addend2)
    OperationPrinting.print_addition(addend1, addend2, sum)

    addend1 = 1.2
    addend2 = 2.4
    sum = Operation.add_two_numbers(addend1, addend2)
    OperationPrinting.print_addition(addend1, addend2, sum)

    addend1 = 2.2
    addend2 = -1.8
    sum = Operation.add_two_numbers(addend1, addend2)
    OperationPrinting.print_addition(addend1, addend2, sum)

    birth_date = date(2004, 11, 10)
    days = PersonController.compute_age_in_days(birth_date)
    years = PersonController.compute_age_in_years(birth_date)

    print(f"David is {days} days old")
    print(f"He was born on {birth_date}")
    print(f"David is {years} years old")

if __name__ == "__main__":
    main()
